### PR TITLE
data: prajapati55

### PR DESCRIPTION
**Pull request by prajapati55**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/3752"></a>

### CODE REVIEW

**Comment by Hrishi5111998** [comment]
hey, your testimonials is empty please add some  into it or remove it. thank you!

File: data/prajapati55.json [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    "Next.js",
+    "Open Source",
+    "Docker",
+    "Node.js"
+  ],
+  "socials": [
+    { "icon": "FaTwitter", "url": "https://twitter.com/pravendra93" },
+    { "icon": "FaGithub", "url": "https://github.com/prajapati55" }
+  ],
+  "testimonials": [],
```
Comment (Panquesito7) [inline data/prajapati55.json:18]: Best to remove this field if it's not used, IMO.
```suggestion
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Review by kumarsonsoff3 — APPROVED** [review]
Looks good, Thanks! 👍

**Pull request merged**